src/routers/people.ts

```diff
@@ -8,24 +8,127 @@ import { StatusCode } from '@/models/statusCodes';
 
 const router = createRouter();
 
+/**
+ * @api {get} /people/
+ * @description Returns all people and all their associated attributes.
+ *              The only guaranteed field is a 'personUuid' and 'category'.
+ *              Parameters have no effect on this request.
+ *              Associated attributes can be checked via /peopleCategories
+ * @apiName GetPeople
+ *
+ * @apiError (401) Unauthorized - Must be admin
+ *
+ * @apiSuccess (200) {String} Successfully retrieved all people
+ * @apiSuccessExample Success-Response:
+  [
+    {
+      "personUuid": "1e167b81-d816-497b-8c0c-36f4d6b2fd33",
+      "category": "farmers",
+      "firstName": "Zachariah",
+      "paymentFrequency": "weekly",
+      "notes": "Brother of Moses",
+    },
+    {
+      "personUuid": "4b3f23a3-04c4-468f-bdf5-f189a34d9f69",
+      "category": "trader",
+      "firstName": "Mary",
+      "username": "maryjoseph9",
+    }
+  ]
+ */
 router.get('/', async (req, res) => {
   res.status(StatusCode.OK).send('Successfully retrieved all people');
 }, authorized(UserType.ADMIN));
 
+/**
+ * @api {get} /people/<category>
+ * @description Returns all people and all their associated attributes.
+ *              Parameters have no effect on this request, but the URL
+ *              specifies what kind of people are given.
+ *              Parameters have no effect on this request.
+ *              Associated attributes can be checked via /peopleCategories
+ * @apiName GetPeopleCategory
+ *
+ * @apiError (401) Unauthorized - Must be admin
+ * @apiError (404) Person category not found
+ *
+ * @apiSuccess (200) {String} Successfully retrieved all people from category <category>
+ * @apiSuccessExample /people/farmers Success-Response:
+  [
+    {
+      "personUuid": "1e167b81-d816-497b-8c0c-36f4d6b2fd33",
+      "firstName": "Zachariah",
+      "paymentFrequency": "weekly",
+      "notes": "Brother of Moses",
+    },
+    {
+      "personUuid": "4b3f23a3-04c4-468f-bdf5-f189a34d9f69",
+      "firstName": "Zachariah",
+      "paymentFrequency": "monthly",
+      "notes": "Has a nice car",
+    }
+  ]
+ */
 router.get('/:category', async (req, res) => {
-  res.status(StatusCode.OK).send('Successfully retrieved all people of type <something>');
+  res.status(StatusCode.OK).send('Successfully retrieved all people of category <something>');
 }, authorized(UserType.ADMIN));
 
+/**
+ * @api {get} /people/<category>
+ * @description Creates a new person in the specified category.
+ *              Returns the UUID created for that person.
+ * @apiName CreatePerson

```

Comment:
in these docs, don't we need to specify what to include in the request? ie. the parameters? like firstName, middleName, lastName, etc?